packages/nimlangserver/package.yaml

```
@@ -0,0 +1,24 @@
+---
+name: nimlangserver
+description: The Nim language server implementation (based on nimsuggest)
+homepage: https://github.com/nim-lang/langserver
+licenses:
+  - MIT
+languages:
+  - Nim
+categories:
+  - LSP
+
+source:
+  # renovate:datasource=github-tags
+  id: pkg:github/nim-lang/langserver@latest
```

I see you've opened an issue with them regarding stable tags. For now let's target commit SHAs so we can get this merged

```suggestion
  # renovate:datasource=git-refs
  id: pkg:github/nim-lang/langserver@f7b32fa651622d77f2da7b357cb152c58fe7b359
```